### ISSUE TITLE
Fix list of queued items

### ISSUE THREAD

**Issue by LaurensRietveld** [LODLaundry/lodlaundry.github.io#115]
The Washing machine has its list of queued items, as well as the basket interface. The basket interface still shows queued items, where the washing machine shows zero. 
On of both is wrong (I assume the washing machine one).

See here for a short version of the basket interface query: http://yasgui.org/short/Vy9ZuX0P

**Comment by wouterbeek** [comment]
Yes this is a known bug. Better to have it documented as such though.
